src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1163,12 +1166,29 @@ def __init__(self,
         self.enable_haproxy_protocol = enable_haproxy_protocol
         self.idmap_conf = idmap_conf
         self.enable_nlm = enable_nlm
+        self.deploy_stunnel = deploy_stunnel
+        self.stunnel_port = stunnel_port or 20490
+        self.stunnel_tls_dir = stunnel_tls_dir
+
+    def validate(self) -> None:
+        super(NFSServiceSpec, self).validate()
+        if self.deploy_stunnel and not self.stunnel_port:
+            raise SpecValidationError(
+                    'Cannot add NFS: deploy_stunnel specified but no stunnel_port specified')
 
     def get_port_start(self) -> List[int]:
         if self.port:
             return [self.port]
         return []
 
+    def get_port_start(self) -> List[int]:
```

Comment:
I stared at this line for a while to make sure I wasn't seeing things, but I think this function is named exactly the same as the one above it.

Reply:
Oops lol yeah. I copied the function and modified it and forgot to remove the original one 🤦‍♂️